default.nix

```diff
@@ -85,6 +85,60 @@ let
       });
     };
   });
+  rawDockerImage = let
+    topologyFile = pkgs.writeText "topology.yaml" ''
+      wallet:
+        relays:
+          [
+            [
+              { host: nodes.awstest.iohkdev.io }
+            ]
+          ]
+        valency: 1
+        fallbacks: 7
+    '';
+    CLUSTER = "mainnet-1.0";
+    SYSTEM_START_TIME = 1504820421;
+    configFiles = pkgs.runCommand "cardano-config" {} ''
+      mkdir -pv $out
```

Comment:
Hmm, could you add a comment why is special derivation needed for configs?

Reply:
if you refer to the `configuration.yaml` as a full path in nix, it lands in `/nix/store/<hash>-configuration.yaml`
then cardano tries to look for the genesis directly in `/nix/store/genesis.json` without any hash, because the `configuration.yaml` said it was in the same dir